docs/scos/dev/guidelines/nfr-guidelines.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Project Development Non-Functional Requirement guidelines
+description: @TODO
+last_updated: @TODO
+template: concept-topic-template
+related:
+- title: Operatable feature guidelines
+  link: docs/scos/dev/guidelines/operatable-feature-guidelines.html
+- title: Process documentation guidelines
+  link: docs/scos/dev/guidelines/process-documentation-guidelines.html
+- title: Monitorable process guidelines
+  link: docs/scos/dev/guidelines/monitorable-process-guidelines.html
+---
+
+
+## Introduction
+
+Non-Functional Requirements (NFRs) are one of the core tools for architects and developers to describe how the system under development supposed to work from technical perspective.
+Defining NFRs can be as important as defining functional requirements, which are usually done by business people (i.e. Product Owners). Missing out on them usually result in
```

Comment:
In general, I find this is as a very good but missing part of the project discovery. Could you also suggest when and how definition of the NFRs need to succeed (f.e. based on aldi best practices)? I found f.e. the following [resource](https://magdamiu.com/2021/02/22/software-architecture-methodology/) that describes how this could be done. Wdyt?

I would also **really like** to keep here our SES initiative in mind - where we offer our customers services on how to start the project best and supervise them throughout their journey. We explain spryker architecture, help to discover a scope, define high-level architecture, etc. IMO adding there NFR definition would be awesome